### ISSUE TITLE
Unable to label: It seems code expects a json file from Google!

### ISSUE THREAD

**Issue by aknirala** [deel-ai/LARD#10]
I tried downloading and labeling a few images as I describe below, while I succeeded in downloading the labeling failed.

For labeling I tried using CLI as:
`python -m src.labeling.generate_dataset test_export_train_dataset.yml`

Contents of test_export_train_dataset.yml is:
```
output_directory: test_FMEP
dataset_name: test_dset_FMEP
included_datasets:
  LARD_train:
    type: earth_studio
    path: test_FMEP/FMEP_test_runway
```
And it exits as:
```
Label export of test_FMEP/FMEP_test_runway/footage/footage.yaml started
Missing data for scenario -- footage ('width' was not found): scenario skipped 
```
It generates the folder structure etc, but .csv file only has header no labels. Also it expected name of .yaml and json file as footage.yaml/json

On debugging, I found the error is originating from class GESDataset(object): in file: src/ges/ges_dataset.py

To my surprise the comments indicate that it is expecting a json file from Google Earth Studio, but I do not have it. It seems instruction in the repository needs to be updated (or likely I am missing something). Here is the comment in ges_dataset.py file:

 ```
def load_data(self):
        """
        Extract the frame parameters which are in a json file generated by GoogleEarth Studio.

        Frame information:
            "cameraFrames":[
                { "position": {"x": ... , "y": ... , "z": ...}
                  "rotation": {"x": ... , "y": ... , "z": ...}
                  "coordinate": {"latitude": ... , "longitude": ... , "altitude": ... }
                  "fovVertical":
                },
                { "position": ....
                },
                ...
            ]
        :return: None

        """
```

Any help/comment is greatly appreciated.

More details:

Step 1: Creating the json file
To test I generated a very simple json file. Code for that is:
```
import json
import numpy as np
from src.ges.ges_dataset import add_or_update_runways
# Common usage: adding a runway to the database
data_file = 'test_runway.json'

airport_name = 'FMEP' # Existing or new airport ICAO code

runway_ids = ['15', '33'] # Supports 1 or 2 runway ID

# Coordinates must be a numpy array of 4 coordinates in WGS84 format:
# ==> Latitude, Longitude, Altitude in meters.
coordinates = np.array([
    [-21.313539792048655, 55.414677867454635, 17 ], # Points 1 and 2 should correspond to the first runway ID
    [-21.313233693596825, 55.414951452761926, 17 ],
    [-21.324679789297647, 55.43004674957136,  14  ], # Points 3 and 4 should correspond to the second runway ID if present,
    [-21.32499113834105,  55.42977957629199,  14  ]  # but these last two points are always required nonetheless
    ])
# This function will add the existing runway to the database or update its coordinates if it exists
add_or_update_runways(data_file, airport_name, runway_ids, coordinates)
```
It created the test_runway.json which is (It is a simple file):

```
{
    "FMEP": {
        "15": {
            "A": {
                "position": {
                    "x": 3367493.6359039284,
                    "y": 4886940.850502305,
                    "z": -2316838.912055465
                },
                "coordinate": {
                    "latitude": -21.324679789297647,
                    "longitude": 55.43004674957136,
                    "altitude": 14.0
                }
            },
            "B": {
                "position": {
                    "x": 3367509.2802003375,
                    "y": 4886914.780692183,
                    "z": -2316871.1622774517
                },
                "coordinate": {
                    "latitude": -21.32499113834105,
                    "longitude": 55.42977957629199,
                    "altitude": 14.0
                }
            },
            "C": {
                "position": {
                    "x": 3369061.595863278,
                    "y": 4886410.45215375,
                    "z": -2315686.0519273253
                },
                "coordinate": {
                    "latitude": -21.313539792048655,
                    "longitude": 55.414677867454635,
                    "altitude": 17.0
                }
            },
            "D": {
                "position": {
                    "x": 3369045.2856945437,
                    "y": 4886436.72434392,
                    "z": -2315654.343085648
                },
                "coordinate": {
                    "latitude": -21.313233693596825,
                    "longitude": 55.414951452761926,
                    "altitude": 17.0
                }
            }
        },
        "33": {
            "A": {
                "position": {
                    "x": 3369061.595863278,
                    "y": 4886410.45215375,
                    "z": -2315686.0519273253
                },
                "coordinate": {
                    "latitude": -21.313539792048655,
                    "longitude": 55.414677867454635,
                    "altitude": 17.0
                }
            },
            "B": {
                "position": {
                    "x": 3369045.2856945437,
                    "y": 4886436.72434392,
                    "z": -2315654.343085648
                },
                "coordinate": {
                    "latitude": -21.313233693596825,
                    "longitude": 55.414951452761926,
                    "altitude": 17.0
                }
            },
            "C": {
                "position": {
                    "x": 3367493.6359039284,
                    "y": 4886940.850502305,
                    "z": -2316838.912055465
                },
                "coordinate": {
                    "latitude": -21.324679789297647,
                    "longitude": 55.43004674957136,
                    "altitude": 14.0
                }
            },
            "D": {
                "position": {
                    "x": 3367509.2802003375,
                    "y": 4886914.780692183,
                    "z": -2316871.1622774517
                },
                "coordinate": {
                    "latitude": -21.32499113834105,
                    "longitude": 55.42977957629199,
                    "altitude": 14.0
                }
            }
        }
    }
}
```

for step 2 I used CLI and only use runway ID 15
for step 3: I uploaded the FMEP_test_runway.esp file to Google Earth Studio and was able to download images.

**Comment by AssuranceCasesEC6** [comment]
Hi Aknirala,

Thank you for using our generator. 
The missing .json file should be the one generated by Google Earth Studio indeed. Could you double-check the instruction [2-bis](https://github.com/deel-ai/LARD?tab=readme-ov-file#2-bis-google-earth-generation-recommendations) ?
Especially this part:
<img width="715" alt="generation_recommandations google earth studio" src="https://github.com/deel-ai/LARD/assets/168717446/1e7eb79d-59a2-4037-9b02-7a230c95f64a">

This should allow Google Earth Studio to produce the appropriate .json file, and you then need to copy your .yaml file in this folder created by Google Earth Studio to obtain the folder structure shown on the image.

Please let me know if it resolves the issue.
Thank you

**Comment by Vincent-Mussot** [comment]
Hi Aknirala,

Thank you for using our generator. 
The missing .json file should be the one generated by Google Earth Studio indeed. Could you double-check the instruction [2-bis](https://github.com/deel-ai/LARD?tab=readme-ov-file#2-bis-google-earth-generation-recommendations) ?
Especially this part:
<img width="715" alt="generation_recommandations google earth studio" src="https://github.com/deel-ai/LARD/assets/168717446/1e7eb79d-59a2-4037-9b02-7a230c95f64a">

This should allow Google Earth Studio to produce the appropriate .json file, and you then need to copy your .yaml file in this folder created by Google Earth Studio to obtain the folder structure shown on the image.

Please let me know if it resolves the issue.
Thank you

**Comment by aknirala** [comment]
Hi @Vincent-Mussot 

Thank you for your prompt response.

Following your instructions in section 2-bis, I was able to generate the .yaml file and label the images accordingly. The missing .json file was indeed created by Google Earth Studio after applying the generation recommendations.

I appreciate your assistance. The issue has been resolved.

Best regards,
Nirala A K
